pyvista/plotting/renderer.py

```diff
@@ -878,10 +878,10 @@ def add_axes(
         ylabel='Y',
         zlabel='Z',
         labels_off=False,
-        marker_args=None,
```

Comment:
Instead of removing this, we should actually deprecate its use.  See https://github.com/pyvista/pyvista/blob/39ef449299eed9b2f5b3ffca419898bffb307bf7/pyvista/core/pointset.py#L204-L207 as an example of how we could do this.  I think if `marker_args` is not `None` then we throw a warning and then add them to **kwargs.